proto/executionapis/astria/execution/v2/executed_block_metadata.proto

```diff
@@ -17,4 +17,7 @@ message ExecutedBlockMetadata {
   // Timestamp of the block, taken from the sequencer block that this rollup block
   // was constructed from.
   google.protobuf.Timestamp timestamp = 4;
+  // The block hash of the sequencer block which this execution block is derived
+  // from. Utilizing this field is optional.
```

Comment:
Nit: we have five variations of this comment 
* The hash of the sequencer block that this block is derived from.
* The block hash of sequencer block this is derived from.
* The hash of the sequencer block the transactions come from.
* The block hash of the sequencer which this block is derived from.
* The block hash of the sequencer block which this execution block is derived from.

I wouldn't have mentioned it except for the fact that "The block hash of the sequencer which this block is derived from." is probably worth changing.

Reply:
It must be optional in order to sync Flame or Forma nodes with the new conductor software.

Cannot change old blocks.

Reply:
This is for execution API v2 though

Reply:
Ah, you mean upgraded flame but old blocks. Ok, that makes sense.

Can we make this required after some block number? Or too much effort?

Reply:
Off line synced, decided yagni.

This is optional from the rollup perspective, it's optional to use it. Conductor will always include it in execution requests, but the execution node doesn't need to use it.